pkg/kubectl/cmd/delete.go

```diff
@@ -166,6 +156,14 @@ func ReapResult(r *resource.Result, f *cmdutil.Factory, out io.Writer, isDefault
 			options = api.NewDeleteOptions(int64(gracePeriod))
 		}
 		if err := reaper.Stop(info.Namespace, info.Name, timeout, options); err != nil {
+			// We intentionally ignore the NotFound error. Because if we reach
+			// here, we must have already validated the existence of the target
+			// object with RetrieveLatest(). If we get a NotFound error here,
+			// it's caused by a race between nodecontroller and apiserver. See
```

Comment:
We're not necessarily deleting pods here, right?

In general, the race would be between kubectl and a controller.